utils/vpr_io_place.py

```diff
@@ -41,7 +41,7 @@ def constrain_net(self, net_name, loc, comment=""):
             comment=comment,
         )
 
-    def output_io_place(self, f):
+    def output_io_place(self, f, net_to_block=None):
```

Comment:
Probably worth adding a docstring explaining what net_to_block is.  Should make a utility function in this python file for generating net_to_block?